lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -271,10 +277,35 @@ bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
         if (TfMapLookup(mNewNodeRegistry, key, &obj)) {
             if (obj.hasFn(MFn::kDagNode)) {
                 addedDagPaths->push_back(MDagPath::getAPathTo(obj));
+                currentAddedDagPaths.append(MDagPath::getAPathTo(obj));
```

Comment:
Can you explain the difference between `addedDagPaths` and `currentAddedDagPaths`?

Reply:
Originally, it looked to me like `addedDagPaths` was being used for purposes other than just acting as a cache of the DAG paths added during the single import job; now, after looking closely, I see that there is no appreciable difference other than the fact that one is `std::vector<MDagPath>`, and the other a `MDagPathArray`. Should import chasers take a parameter with the signature of the former, or latter (since they're not really equivalent under the hood?)

Reply:
Since we are using Maya types here, let's stick to MDagPathArray.